feature/aft/aft_base/otg_tests/aft_prefixcounters/README.md

```diff
@@ -4,162 +4,199 @@
 
 IPv4/IPv6 prefix counters
 
-## Procedure
+## Test Setup
+
+### Generate DUT and ATE Configuration
 
 Configure DUT:port1 for IS-IS session with ATE:port1
 *   IS-IS must be level 2 only with wide metric.
 *   IS-IS must be point to point.
 *   Send 1000 ipv4 and 1000 ipv6 IS-IS prefixes from ATE:port1 to DUT:port1.
-*   IS-IS must be up and running, prefixes must be in RIB and FIB.
 
 Establish eBGP sessions between ATE:port1 and DUT:port1.
 *   Configure eBGP over the interface ip.
 *   Advertise 1000 ipv4,ipv6 prefixes from ATE port1 observe received prefixes at DUT.
-*   Validate total number of entries of AFT for IPv4 and IPv6.
-*   Each prefix must have 1 next hop pointing to ATE port1.
 
-Verifications
+## Procedure
```

Comment:
This looks like a subtest here.  Please define a heading for it.  (ie: this should be AFT-2.1.1, and the following subtest numbering should be updated to follow)

Reply:
done. Added as part of test set up.